client/components/NavBar/NavBar.jsx

```diff
@@ -19,23 +38,38 @@ class Nav extends React.Component {
           <h1>
             <Link to="/" className={logo}>Beer.ly</Link>
           </h1>
-          <h1>
-            <Link to="/browse">Browse</Link>
-          </h1>
-          <h1>
-            <Link to="/">Ratings</Link>
-          </h1>
-          <h1>
-            <Link to="/">Recommendations</Link>
-          </h1>
-          <h1>
-            <Link to="/">Delivery</Link>
-          </h1>
-          <ul>
-            <li>
-              {cart}
-            </li>
-          </ul>
+          { !isHomePage ?
```

Comment:
Can potentially wrap all of the conditional rendering under one {!isHomePag ? ... : ... } conditional rendering statements instead of many across each individual <Link>.